bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/config/POP3IMAPConfig.java

```diff
@@ -23,4 +23,5 @@
 @NonNullByDefault
 public class POP3IMAPConfig extends BaseConfig {
     public int refresh = 60;
+    public Boolean starttls = Boolean.FALSE;
```

Comment:
According to the documentation, enabling starttls will also work for servers that do not support it (it isn‘t used then). So I don‘t think we need a configuration option for that. We can just enable it for all connections.